### PR TITLE
Add README for posix-time library

### PR DESCRIPTION
**Pull request by RaduNichita**
### Prerequisite checklist

 - [x] Read the [contribution guidelines](https://unikraft.org/docs/contributing/) regarding submitting new changes to the project;
 - [x] Tested your changes against relevant architectures and platforms;
 - [x] Ran the [`checkpatch.uk`](https://github.com/unikraft/unikraft/blob/staging/support/scripts/checkpatch.uk) on your commit series before opening this PR;
 - [x] Updated relevant documentation.

### Base target

 - Architecture(s): N/A
 - Platform(s):  N/A
 - Application(s): N/A

### Description of changes

Add README for `posix-time` internal library.

### CODE REVIEW

**Review by StefanJum — CHANGES_REQUESTED** [review]
Thanks @RaduNichita, please add a short application example that uses some `posix-time` functions and short instructions on how to enable it from the menuconfig screen.

**Comment by RaduNichita** [comment]
@StefanJum I have applied the suggestions. Please have another look when you have time.

File: lib/posix-time/README.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# posix-time
+
+`posix-time` is an internal library of Unikraft that enables the system calls that are dependent of the system clock.
+
+Note: the operation with timers are not yet supported.
```
Comment (StefanJum) [inline lib/posix-time/README.md]:
```suggestion
Note: timer operations are not yet supported.
```

File: lib/posix-time/README.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+You can select `posix-time` under the `Library Configuration` screen of the `make menuconfig` command.
+After that, you can use the functions exposed by the internal library just like you would for a Linux system,
+
+An example of a simple application that uses the `posix-time` library is the following:
+
+```c
+#include <stdio.h>
+#include <time.h>
+
+int main(void) {
```
Comment (StefanJum) [inline lib/posix-time/README.md]:
```suggestion
int main(void) {
```

**Review by StefanJum — CHANGES_REQUESTED** [review]
Few comments @RaduNichita, also sign-off the commit.

**Comment by RaduNichita** [comment]
I applied the changes. @StefanJum please have a look when you have time.

**Review by razvand — APPROVED** [review]
Reviewed-by: Razvan Deaconescu <razvand@unikraft.io>
Approved-by: Razvan Deaconescu <razvand@unikraft.io>

**Pull request merged**